README.md

```diff
@@ -68,92 +69,106 @@ To try out the [examples](https://github.com/iotaledger/identity.rs/blob/HEAD/ex
 
 ## Example: Creating an Identity
 
-The following code creates and publishes a new IOTA DID Document to the Tangle Mainnet. 
+The following code creates and publishes a new IOTA DID Document to the Tangle Mainnet.
+
+_Cargo.toml_
 
-*Cargo.toml*
 ```toml
 [package]
 name = "iota_identity_example"
 version = "1.0.0"
 edition = "2021"
 
 [dependencies]
-identity_iota = { version = "0.6" }
+identity_iota = { version = "0.7" }
+iota-client = { version = "2.0.0-beta.3", default-features = false, features = ["tls", "stronghold"] }
 tokio = { version = "1", features = ["full"] }
 ```
-*main.*<span></span>*rs*
+
+_main._<span></span>_rs_
+
 ```rust,no_run
-use identity_iota::account::Account;
-use identity_iota::account::IdentitySetup;
-use identity_iota::account::Result;
-use identity_iota::account_storage::Stronghold;
 use identity_iota::core::ToJson;
-use identity_iota::client::ExplorerUrl;
-use identity_iota::client::ResolvedIotaDocument;
-
+use identity_iota::crypto::KeyPair;
+use identity_iota::crypto::KeyType;
+use identity_iota::did::MethodScope;
+use identity_iota::iota_core::IotaClientExt;
+use identity_iota::iota_core::IotaDocument;
+use identity_iota::iota_core::IotaIdentityClientExt;
+use identity_iota::iota_core::IotaVerificationMethod;
+use identity_iota::iota_core::NetworkName;
```

Comment:
You mean `identity_iota::{IotaDocument, IotaDID, ... }`? What do we do if we add a second DID method, will that then also be re-exported from the top or in a module? Personally, I think having types in modules is more consistent with the rest of the crate.
Why do you think we should re-export it from the top?

Reply:
The top-level crate is called `identity_iota`, the entire library is about the IOTA DID method. Other methods would still be in modules.

Reply:
I see `identity_iota` as standing on for "IOTA Identity: DID methods building on IOTA" which is not exclusively the UTXO method, or at least isn't planned to be.

Keeping the modules is also much nicer for the top-level crate documentation. Currently it is:

> ## Modules
>
> core Core Traits and Types
> credential Verifiable Credentials
> crypto Cryptographic Utilities
> did Decentralized Identifiers
> iota_core The IOTA DID method implementation for the IOTA ledger.
> prelude Prelude of commonly used types
> resolver DID resolution utilities 

With the re-export it would be:

> ## Modules
> 
> core Core Traits and Types
> credential Verifiable Credentials
> crypto Cryptographic Utilities
> did Decentralized Identifiers
> iota_core The IOTA DID method implementation for the IOTA ledger.
> prelude Prelude of commonly used types
> resolver DID resolution utilities
> 
> ## Structs
> 
> IotaDID A DID conforming to the IOTA DID method specification.
> IotaDocument A DID Document adhering to the IOTA DID method specification.
> IotaDocumentMetadata Additional attributes related to a IotaDocument.
> NetworkName Network name compliant with the crate::IotaDID method specification.
> 
> ## Enums
> Error
> StateMetadataEncoding Indicates the encoding of a DID document in state metadata.
> 
> ## Type Definitions
> 
> IotaCoreDocument A CoreDocument whose fields adhere to the IOTA DID method specification.
> IotaDIDUrl A DID URL conforming to the IOTA DID method specification.
> IotaService A Service adhering to the IOTA DID method specification.
> IotaVerificationMethod A VerificationMethod adhering to the IOTA DID method specification.
> Result

which to developers using the second method, would include types they don't care about and is a more messy entry point in general.

Since we don't expect the UTXO method to be strictly superior to the second method, I feel like opinionating the library towards the UTXO method doesn't make sense.

So I'm still in favor of keeping things modularized.

Reply:
Having multiple DID methods on IOTA is a good reason to keep them modularized. Can we perhaps change the `iota_core` module to `iota` at least, so it matches the method name? There is no longer an `iota_core`, `iota_client` split.